packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -787,10 +777,6 @@ export abstract class FluidDataStoreContext
 	 */
 	public makeLocallyVisible() {
 		assert(this.channel !== undefined, 0x2cf /* "undefined channel on datastore context" */);
-		assert(
```

Comment:
Why is this assert removed?

Reply:
https://github.com/microsoft/FluidFramework/pull/19691/files#r1505064353 😄